backend/views/syson-diagram-statetransition-view/src/main/java/org/eclipse/syson/diagram/statetransition/view/services/StateTransitionViewNodeToolSectionSwitch.java

```diff
@@ -79,43 +83,44 @@ protected List<NodeDescription> getAllNodeDescriptions() {
 
     @Override
     public List<NodeToolSection> caseDefinition(Definition object) {
-        var createSection = this.buildCreateSection();
+        var createSection = this.toolDescriptionService.buildCreateSection();
         createSection.getNodeTools().addAll(this.createToolsForCompartmentItems(object));
         return List.of(createSection);
     }
 
     @Override
     public List<NodeToolSection> caseStateDefinition(StateDefinition object) {
-        var createSection = this.buildCreateSection(
+        var createSection = this.toolDescriptionService.buildCreateSection(
                 new StateTransitionCompartmentNodeToolProvider(false, false).create(null),
                 new StateTransitionCompartmentNodeToolProvider(true, false).create(null),
                 new StateTransitionCompartmentNodeToolProvider(false, true).create(null),
                 new StateTransitionCompartmentNodeToolProvider(true, true).create(null),
-                new StateTransitionActionToolProvider(SysmlPackage.eINSTANCE.getStateDefinition_EntryAction()).create(null),
-                new StateTransitionActionToolProvider(SysmlPackage.eINSTANCE.getStateDefinition_DoAction()).create(null),
-                new StateTransitionActionToolProvider(SysmlPackage.eINSTANCE.getStateDefinition_ExitAction()).create(null),
-                new CompartmentNodeToolProvider(SysmlPackage.eINSTANCE.getElement_Documentation(), this.descriptionNameGenerator).create(null));
```

Comment:
Why do you have deleted the documentation tool?

Reply:
I have never done this :P Fixed, rebase mistake. Thanks for noticing.